app/managers.py

```diff
@@ -0,0 +1,40 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema_db.sqlite")
+        self._table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self._table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list:
+        actor_cursor = self._connection.execute(
+            "SELECT * FROM {self._table_name}",

```

Comment:
The table name in the SQL query should be formatted correctly. Use f-string formatting: `f"SELECT * FROM {self._table_name}"`.